package/test_requirements.txt

```diff
@@ -8,6 +8,6 @@ behave>=1.2.6, <2.0
 pylint>=2.4.4, <2.5.0 # due to a regression in 2.5.0: https://github.com/PyCQA/pylint/issues/3524
 pytest-mock>=1.7.1,<2.0
 requests-mock>=1.6.0, <2.0
-flake8>=3.5, <4.0
+flake8>=3.5, <3.8.1
```

Comment:
<3.8.0 since it was released 20 hours ago?

Reply:
I'm trying them one by one. Who knows which break it